selfdrive/ui/qt/offroad/developer_panel.cc

```diff
@@ -37,7 +45,6 @@ void DeveloperPanel::updateToggles(bool _offroad) {
     btn->setEnabled(_offroad);
   }
 
-  // longManeuverToggle should not be toggleable if the car don't have longitudinal control
```

Comment:
What is the reason to delete this coment?

Reply:
My mistake, I'll re-add the comment.